app/controllers/catalog_controller.rb

```diff
@@ -2122,7 +2122,7 @@ def x_edit_tags_reset(db)
       session[:tag_db] = @tagging = params[:tagging] if params[:tagging]
     end
 
-    @gtl_type = 'list' # No quad icons for user/group list views
+    @gtl_type = 'grid' # No quad icons for user/group list views
```

Comment:
@jzigmund  Pls remove the comment, it is irrelevant after changing gtl_type assignment.